diff --git a/node_modules/react-native-maps/ios/AirMaps/AIRMap.m b/node_modules/react-native-maps/ios/AirMaps/AIRMap.m
index fe1ae13..af863d9 100644
--- a/node_modules/react-native-maps/ios/AirMaps/AIRMap.m
+++ b/node_modules/react-native-maps/ios/AirMaps/AIRMap.m
@@ -116,13 +116,13 @@ - (void)insertReactSubview:(id<RCTComponent>)subview atIndex:(NSInteger)atIndex
         [self addOverlay:(id<MKOverlay>)subview];
     } else if ([subview isKindOfClass:[AIRMapUrlTile class]]) {
         ((AIRMapUrlTile *)subview).map = self;
-        [self addOverlay:(id<MKOverlay>)subview];
+        [self addOverlay:(id<MKOverlay>)subview level:MKOverlayLevelAboveLabels];
     }else if ([subview isKindOfClass:[AIRMapWMSTile class]]) {
         ((AIRMapWMSTile *)subview).map = self;
         [self addOverlay:(id<MKOverlay>)subview];
     } else if ([subview isKindOfClass:[AIRMapLocalTile class]]) {
         ((AIRMapLocalTile *)subview).map = self;
-        [self addOverlay:(id<MKOverlay>)subview];
+        [self addOverlay:(id<MKOverlay>)subview level:MKOverlayLevelAboveLabels];
     } else if ([subview isKindOfClass:[AIRMapOverlay class]]) {
         ((AIRMapOverlay *)subview).map = self;
         [self addOverlay:(id<MKOverlay>)subview];
