diff --git a/node_modules/react-native-elements/dist/searchbar/SearchBar-android.js b/node_modules/react-native-elements/dist/searchbar/SearchBar-android.js
index fe25d36..d859013 100644
--- a/node_modules/react-native-elements/dist/searchbar/SearchBar-android.js
+++ b/node_modules/react-native-elements/dist/searchbar/SearchBar-android.js
@@ -45,13 +45,13 @@ class SearchBar extends Component {
     constructor(props) {
         super(props);
         this.focus = () => {
-            this.input.focus();
+            this.input?.focus?.();
         };
         this.blur = () => {
-            this.input.blur();
+            this.input?.blur?.();
         };
         this.clear = () => {
-            this.input.clear();
+            this.input?.clear?.();
             this.onChangeText('');
             this.props.onClear();
         };
