diff --git a/node_modules/react-native-gifted-chat/lib/Day/index.js b/node_modules/react-native-gifted-chat/lib/Day/index.js
index b9719b6..e068513 100644
--- a/node_modules/react-native-gifted-chat/lib/Day/index.js
+++ b/node_modules/react-native-gifted-chat/lib/Day/index.js
@@ -21,7 +21,7 @@ export function Day({ dateFormat = DATE_FORMAT, dateFormatCalendar, createdAt, c
             return date.format('D MMMM YYYY');
         if (now.diff(date, 'days') < 1)
             return date.calendar(now, {
-                sameDay: '[Today]',
+                sameDay: '[Heute]',
                 ...dateFormatCalendar,
             });
         return date.format(dateFormat);
