diff --git a/node_modules/react-native-elements/dist/searchbar/SearchBar-android.js b/node_modules/react-native-elements/dist/searchbar/SearchBar-android.js
index fe25d36..d859013 100644
--- a/node_modules/react-native-elements/dist/searchbar/SearchBar-android.js
+++ b/node_modules/react-native-elements/dist/searchbar/SearchBar-android.js
@@ -45,13 +45,13 @@ class SearchBar extends Component {
     constructor(props) {
         super(props);
         this.focus = () => {
-            this.input.focus();
+            this.input?.focus?.();
         };
         this.blur = () => {
-            this.input.blur();
+            this.input?.blur?.();
         };
         this.clear = () => {
-            this.input.clear();
+            this.input?.clear?.();
             this.onChangeText('');
             this.props.onClear();
         };
diff --git a/node_modules/react-native-elements/dist/list/ListItemBase.js b/node_modules/react-native-elements/dist/list/ListItemBase.js
index 03b9a57..b4c1f73 100644
--- a/node_modules/react-native-elements/dist/list/ListItemBase.js
+++ b/node_modules/react-native-elements/dist/list/ListItemBase.js
@@ -39,7 +39,7 @@ const PadView = (_a) => {
     return (<Container {...props} ref={_root} testID="padView">
       {React.Children.map(childrens, (child, index) => child && [
             child,
-            index !== length - 1 && <View style={{ paddingLeft: pad }}/>,
+            index !== length - 1 && <View key={`pad-spacer-${index}`} style={{ paddingLeft: pad }}/>,
         ])}
     </Container>);
 };
